index.js

```diff
@@ -60,15 +60,53 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+function toggleColor() {
+  if (event.target.style.color){
+    if (event.target.style.color === "red") {
+      event.target.style.color = "black"
+    } else {
+      event.target.style.color = "red"
+    }
   } else {
-    element.style.color = "red"
+    event.target.style.color = "red"
   }
 }
 
+const h1 = document.querySelector("#header")
+h1.addEventListener("click", toggleColor)
```

Comment:
This works, but for a kind of strange reason (the browser creates a global `event` object when it's executing an event callback). You should ideally define the function to take an `event` parameter instead of relying on the global event object:

```js
function toggleColor(event) {
    if (event.target.style.color === "red") {
      event.target.style.color = "black"
    } else {
      event.target.style.color = "red"
    }
}
```